eclass/kernel-build.eclass

```diff
@@ -104,8 +104,8 @@ IUSE="+strip"
 # @DESCRIPTION:
 # If KERNEL_IUSE_GENERIC_UKI is set, this variable allows setting the
 # built-in kernel command line for the UKI. If unset, the default is
-# root=/dev/gpt-auto-root ro
-: "${KERNEL_GENERIC_UKI_CMDLINE:=root=/dev/gpt-auto-root ro}"
+# root=/dev/gpt-auto-root ro lockdown=integrity
+: "${KERNEL_GENERIC_UKI_CMDLINE:=root=/dev/gpt-auto-root ro lockdown=integrity}"
```

Comment:
Doesn't this effectively force all generic-uki users to set signed modules up, even if they're not using secure boot?

Reply:
> Doesn't this effectively force all generic-uki users to set signed modules up, even if they're not using secure boot?

Well no, because you can override the kernel command line if secure boot is disabled. But yes, there is an extra step required if you want to use unsigned external modules.

So we have four different possible setups, if we don't add `lockdown=integrity`, i.e. what we have now:
- secure boot on, lockdown on: impossible to achieve since we cannot change the cmdline to enable lockdown
- secure boot off, lockdown on: possible, by adding `lockdown=integrity` to the cmdline
- secure boot on, lockdown off: no chances required
- secure boot off, lockdown off: no chances required

After adding `lockdown=integrity` we have instead:
- secure boot on, lockdown on: no chances required
- secure boot off, lockdown on: no changes requied
- secure boot on, lockdown off: impossible to achieve since we cannot change the cmdline to enable lockdown
- secure boot off, lockdown off: possible, by removing `lockdown=integrity` from the cmdline 

In both cases we have one situation that we cannot reach, but I would argue that the latter makes more sense. And then if users want to use external modules without signing them, they will have to disable secureboot and remove  `lockdown=integrity` from the cmdline. Disabling secure boot to use unsigned modules is also what would be required if we opt instead for solution 3 to Bug 921195.